packages/patterns-react/src/patterns/FeaturedLink/FeaturedLinkItem.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import { Button } from 'carbon-components-react';
+import { ArrowRight20 } from '@carbon/icons-react';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Card Array Item Object
+ *
+ * @param {object} props props object
+ * @param {string} props.title Card Title
+ * @param {string} props.copy Card copy
+ * @param {object} props.link Card link object
+ * @returns {*} JSX Card Array Item Object
+ */
+const FeaturedLinkItem = ({ title, copy, link }) => {
+  return (
+    <div
+      data-autoid={`${stablePrefix}--featuredlink-item`}
+      className={`${prefix}--featuredlink-item`}>
+      <h4 className={`${prefix}--featuredlink-item__title`}>{title}</h4>
+      <h2 className={`${prefix}--featuredlink-item__content`}>{copy}</h2>
+      <div className={`${prefix}--featuredlink-item__link`}>
+        <Button
+          className={`${prefix}--featuredlink-item__link__button`}
+          href={link.href}
+          target={link.target}
+          kind="primary">
+          <ArrowRight20 fill="white" />
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+FeaturedLinkItem.propTypes = {
+  title: PropTypes.string,
```

Comment:
are any of these required?
@photodow is there a content model we need to be following here?

Reply:
I would leverage the CardLink here instead of creating a new "CardLink". I'm still working on it, but it should be similar to what you see here for FeaturedLink's functional specs.

![image](https://user-images.githubusercontent.com/3793636/71007908-5ecac700-20ad-11ea-9124-b7f1789ac0a4.png)
https://ibm.box.com/s/tlv7h1zgc886l5ovcap08uc1x97jpdo2